jenkins/scripts/perf/README.md

```
@@ -1,64 +1,110 @@
+# Perf Sanity Scripts
 
-This directory contains `perf_sanity_triage.py`, a helper script for querying
-and updating perf sanity data in OpenSearch, and for sending regression
-summaries to Slack.
+This directory contains scripts for running perf sanity tests and managing perf sanity data.
 
-## Basic Usage
+## Directory Structure
 
-This script is run by the Jenkins pipeline. Inputs are configured in `jenkins/runPerfSanityTriage.groovy`:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)